import_flow_events.py

```diff
@@ -54,22 +59,63 @@
       utmTerm VARCHAR(40)
     );
 """
+Q_CREATE_METADATA_TABLE = """
+    CREATE TABLE IF NOT EXISTS flow_metadata (
+      flowId VARCHAR(64) NOT NULL UNIQUE SORTKEY,
+      beginTime TIMESTAMP NOT NULL,
```

Comment:
What's the advantage of using`flowId` for the `sortkey`?  An alternative might be to make it the `distkey`, and use `beginTime` as the `sortkey`, to let us to time-based summary queries from this table. (like "number of compelete flows per day, or time").